config/settings/development.yml

```diff
@@ -1,6 +1,7 @@
-host_inventory_url: https://insights-inventory-platform-ci.ask-on-slack-for-the-inventory-devel-url
-rbac_url: https://insights-inventory-platform-ci.ask-on-slack-for-the-rbac-devel-url
-#prometheus_exporter_host: prometheus
-#prometheus_exporter_port: 9394
+host_inventory_url: http://inventory-web:8081
+remediations_url: https://ci.cloud.redhat.com
+rbac_url: https://ci.cloud.redhat.com
```

Comment:
idk how bad this is, but if we want it to work out of the box, this is required.